src/js/components/SelectMultiple/stories/typescript/Children.tsx

```diff
@@ -0,0 +1,109 @@
+import React, { useState } from 'react';
+
+import { FormClose } from 'grommet-icons';
+import { Box, Button, Text } from 'grommet';
+import { SelectMultiple } from '../../SelectMultiple.js';
+
+const allSeasons = [
+  'S01',
+  'S02',
+  'S03',
+  'S04',
+  'S05',
+  'S06',
+  'S07',
+  'S08',
+  'S09',
+  'S10',
+];
+
+export const Children = () => {
+  const [selected, setSelected] = useState([]);
+
+  const onRemoveSeason = (season) => {
+    setSelected(selected.filter((selectedSeason) => selectedSeason !== season));
+  };
+
+  const renderSeason = (season) => (
+    <Button
+      key={`season_tag_${season}`}
+      onClick={(event) => {
+        event.preventDefault();
+        event.stopPropagation();
+        onRemoveSeason(season);
+      }}
+      onFocus={(event) => event.stopPropagation()}
+      a11yTitle={season}
+    >
+      <Box
+        align="center"
+        direction="row"
+        gap="xsmall"
+        pad={{ vertical: 'xsmall', horizontal: 'small' }}
+        margin="xsmall"
+        background="brand"
+        round="large"
+      >
+        <Text size="small">{season}</Text>
+        <Box round="full" margin={{ left: 'xsmall' }}>
+          <FormClose size="small" style={{ width: '12px', height: '12px' }} />
+        </Box>
+      </Box>
+    </Button>
+  );
+
+  const renderOption = (option, state) => (
+    <Box pad="small" background={state.active ? 'active' : undefined}>
+      {option}
+    </Box>
+  );
+
+  return (
+    // Uncomment <Grommet> lines when using outside of storybook
+    // <Grommet theme={...}>
+    <Box fill align="center" pad="small">
+      <SelectMultiple
```

Comment:
This example looks odd with the `0 of 10 selected` text when nothing is selected and wasn't clear, especially with the placeholder being the default text color

Reply:
I changed this so that when nothing is selected the `0 out of x` doesn't show and instead we just see the placeholder or valueLabel